src/visualization/types/Graph/view.tsx

```diff
@@ -174,6 +174,12 @@ const XYPlot: FC<Props> = ({properties, result, timeRange, annotations}) => {
     return <EmptyGraphMessage message={INVALID_DATA_COPY} />
   }
 
+  const editAnnotation = (userSelectedAnnotation: Annotation) => {
+    event('xyplot.annotations.edit_annotation.edit')
+    /* eslint-disable no-console */
+    console.log('edit!', userSelectedAnnotation)
```

Comment:
✂️

Reply:
I want to leave this here to have the signature stay the same and to prevent me from having to add more typescript rules just to leave the unused var in place lol - you can thank Typescript's obnoxious yelling